root/io/uniquePointer/ns_privacy.hh

```diff
@@ -0,0 +1,50 @@
+#ifndef ns_privacy_hh
```

Comment:
nitpick, but usually I see headers with the `.hxx` extension, at least for new code.